src/main/webapp/resources/js/contexts/AnalysisContext.js

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from "react";
+import { showNotification } from "../modules/notifications";
+
+import { updateAnalysis } from "../apis/analysis/analysis";
+
+const stateMap = {
+  NEW: 0,
+  PREPARING: 1,
+  SUBMITTING: 2,
+  RUNNING: 3,
+  FINISHED_RUNNING: 3,
+  POST_PROCESSING: 4,
+  TRANSFERRING: 4,
+  COMPLETING: 4,
+  COMPLETED: 5
+};
+
+const initialContext = {
+  analysis: window.PAGE.analysis,
+  analysisName: window.PAGE.analysisName,
+  analysisState: window.PAGE.analysisState,
+  analysisType: window.PAGE.analysisType,
+  isAdmin: window.PAGE.isAdmin,
+  stateMap: stateMap,
```

Comment:
Does the `stateMap` need to be stored in state?  Is it something that can change?  Remember, only store stuff in state that should be changing.

Reply:
Along with `isAdmin`

Reply:
Fixed in a4521e1c138b57a39b829edd7036468c87580998